source/_components/light.deconz.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "deCONZ lights"
+description: "Instructions how to integrate your Zigbee lights from deCONZ into Home Assistant."
+date: 2017-11-12 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: zigbee.png
+ha_category: Light
+ha_release: "0.58"
+ha_iot_class: "Local Push"
+---
+
+See the [deCONZ main component](/components/deconz/) for configuration instructions.
+
+Lights entity names will be light.device_name, where device_name comes from deCONZ.
+
+#### {% linkable_title Verified to be supported sensors %}
+
+- IKEA Trådfri bulb GU10 W 400lm
```

Comment:
Tested "TRADFRI bulb E27 WS�opal 980lm" as working. Not sure about the weird character in there but thats how it displays in the front end (and probably deconz to be fair)

Reply:
Hmm, maybe I need to decode it to a specific type like UTF-8?